### PR TITLE
MIMP-12 rio upload

### PR DESCRIPTION
**Pull request by JanionNevill**
Injected data page from MetaPairwise. Some further tidying-up to fix an error when running the frequentist analysis.

This needs to wait until after PR #10 has been merged, as this branch is based on that one.

### CODE REVIEW

**Comment by tommorris168** [comment]
Am I supposed to be reviewing only the last three commits? Those are the only ones that seem relevant to this task.
Also there is no dev branch.

**Comment by JanionNevill** [comment]
> Am I supposed to be reviewing only the last three commits? Those are the only ones that seem relevant to this task. Also there is no dev branch.

Yes, last 3 commits only. Thanks Tom :)

File: tests/testthat/helper-formatting.R [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+#' Create a formatted string for a vector. This is to be used in test method where using `!!` fails to give a useful output.
+#' 
+#' @param vector Vector to display
+#' @return The formatted string for the vector
+format_vector_to_string <- function(vector) {
+  is_char = is.character(vector)
```
Comment (tommorris168) [inline tests/testthat/helper-formatting.R:7]: These = should all be <-

**Review by tommorris168 — CHANGES_REQUESTED** [review]
General comment: on the Data tab, the example datasets on the bottom left are ordered as continuous then binary, whereas in the format requirements on the right they are ordered as binary then continuous. One could be changed for consistency.

**Comment by tommorris168** [comment]
One more thing: you should add yourself as an author.

**Review by tommorris168 — APPROVED** [review]
Yes

**Review by tommorris168 — APPROVED** [review]
Yes?

**Comment by tommorris168** [comment]
For some reason when I try to accept the changes it is just adding a comment.

**Comment by tommorris168** [comment]
Also, there isn't a "resolve conversation" option for my comments. I don't know what I should be doing for that to appear.

**Pull request merged**